src/Resources/Resources.Test/ScenarioTests/DeploymentTests.cs

```diff
@@ -155,5 +155,19 @@ public void TestNewDeploymentWithQueryString()
         {
             TestRunner.RunTestScript("Test-NewDeploymentWithQueryString");
         }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.LiveOnly)]
```

Comment:
It's good to add scenario tests but what I meant for **unit test** was something like:
```csharp
[Fact]
[Trait(Category.AcceptanceType, Category.CheckIn)]
public void TestBicepUtility()
{
    Except(BicepUtility.IsBicepFile("C:\\abc.bicep")).ToBe(true);
    Except(BicepUtility.IsBicepFile("C:\\abc.json")).ToBe(false);
}
```